### PR TITLE
Add metadata symbol lookups

### PR DESCRIPTION
**Pull request by MichaelClerx**
Part of #39

### CODE REVIEW

**Comment by MichaelClerx** [comment]
@jonc125 @tamuri I think this is ready for review now?

File: cellmlmanip/model.py [hunk 1]
```diff
@@ -567,6 +568,65 @@ def get_symbol_by_cmeta_id(self, cmeta_id):
 
         raise KeyError('No variable with cmeta id "%s" found.' % str(cmeta_id))
 
+    def get_symbol_by_ontology_term(self, namespace, local_name):
+        """
+        Searches the RDF graph for a variable annotated with the given
+        ``namespace:local_name`` and returns its symbol.
```
Comment (jonc125) [inline cellmlmanip/model.py]: `namespace` here isn't a prefix, so this doc is potentially confusing. I'd call the parameter `namespace_uri` and describe this as `{namespace_uri}local_name` which is a standard representation; similarly in the next para and error messages below.

File: cellmlmanip/model.py [hunk 2]
```diff
@@ -567,6 +568,65 @@ def get_symbol_by_cmeta_id(self, cmeta_id):
...
+
+        Specifically, this method searches for a unique variable annotated with
+        predicate ``http://biomodels.net/biology-qualifiers/is`` and the object
+        specified by ``namespace + local_name``.
+
+        Will raise a ``KeyError`` if no variable with the given annotation is
+        found, and a ``RuntimeError`` if more than one variable with the given
```
Comment (jonc125) [inline cellmlmanip/model.py]: `RuntimeError` is a bit generic. Would `ValueError` be better?

File: cellmlmanip/rdf.py [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+        # Create blank node
+        node = rdflib.BNode()
+
+    return node
+'''
+
+
+def create_rdf_node(namespace, local_name):
+    """
+    Creates and returns an RDF node from the given ``ns_``,
```
Comment (jonc125) [inline cellmlmanip/rdf.py]: Mismatched name

File: tests/test_rdf.py [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+        model.get_symbol_by_ontology_term(oxmeta, 'time')
+    assert 'Multiple variables annotated with' in str(e)
+
+    # Annotation with a cmeta id that doesn't exist
+    with pytest.raises(KeyError) as e:
+        model.get_symbol_by_ontology_term(oxmeta, 'membrane_potential')
+    assert 'No variable with cmeta id' in str(e)
+
+    # Annotation of something that isn't a variable
+    # TODO: At the moment this fails because there is no variable called 'c'.
```
Comment (jonc125) [inline tests/test_rdf.py]: I think failing like this is fine: it's correct to say that there isn't a variable with this id, after all!

File: cellmlmanip/model.py (hunk 2 above)
Comment (MichaelClerx) [reply cellmlmanip/model.py]: I suppose... If it's legal to have multiple things annotated with "is", but not to request one and insist it's unique...

File: cellmlmanip/rdf.py (hunk 3 above)
Comment (MichaelClerx) [reply cellmlmanip/rdf.py]: Thanks!

File: tests/test_rdf.py (hunk 4 above)
Comment (MichaelClerx) [reply tests/test_rdf.py]: ok!

File: cellmlmanip/model.py (hunk 2 above)
Comment (jonc125) [reply cellmlmanip/model.py]: It's slightly tenuous, but I don't think would surprise users!

File: cellmlmanip/model.py (hunk 1 above)
Comment (MichaelClerx) [reply cellmlmanip/model.py]: Done!

**Pull request merged**